wit-manifest.json

```
@@ -1,7 +1,7 @@
 [
     {
         "//": "v0.2.0",
```

```suggestion
        "//": "v0.2.1",
```

Going to update this comment to say v0.2.1 by live editing on GitHub itself.